src/log/memory_backend.cr

```diff
@@ -0,0 +1,10 @@
+# A `Log::Backend` suitable for testing. It is registered as `memory`.
+#
+@[Log::RegisterBackend("memory")]
+class Log::MemoryBackend < Log::Backend
```

Comment:
I'm sure the annotation name is carefully chosen but why not just use a simpler one:

```crystal
@[Log::Backend("memory")]
class Log::MemoryBackend < Log::Backend
```

Reply:
Well, there's already a class named like that...